apps/finance/test/finance.js

```diff
@@ -620,44 +591,44 @@ contract('Finance App', ([root, owner, recipient]) => {
         // Normal payments
         await assertRevert(
           finance.newImmediatePayment(token1.address, recipient, 10, ''),
-          errors.FINANCE_COMPLETE_TRANSITION
+          ERRORS.FINANCE_COMPLETE_TRANSITION
         )
 
         // Direct ETH transfers
-        await assertRevert(finance.send(10, { gas: 3e5 }), errors.FINANCE_COMPLETE_TRANSITION)
+        await assertRevert(finance.send(10, { gas: 3e6 }), ERRORS.FINANCE_COMPLETE_TRANSITION)
```

Comment:
A 10x increase in gas is quite a lot for this case, although I'm not sure how the previous test was passing (3e5 gas doesn't sound like it'd be enough to fail period transitions!).